scalability_and_performance/understanding-node-observability-operator.adoc

```diff
@@ -0,0 +1,81 @@
+:_content-type: ASSEMBLY
+[id="using-node-observability-operator"]
+= Understanding the Node Observability Operator
+include::_attributes/common-attributes.adoc[]
+:context: node-observability-operator
+
+toc::[]
+
+Node Observability Operator is an optional add-on Operator that provides a link:https://cri-o.io/#what-is-cri-o[CRI-O] and link:https://kubernetes.io/docs/concepts/overview/components/#kubelet[kubelet] profiling on selected worker nodes.
+
+== Key features
+* Trigger a CRIO-O and kublet profiling.
+* You can deploy Node Observability Agents on selected nodes.
+* Users can access the profiling data files for analysis.
+
+== Installing the Node Observability Operator using the web console
+The Node Observability Operator is not installed in {product-title} by default. You can install the Node Observability Operator by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the OpenShift Container Platform web console.
+. Navigate to *Operators* → *OperatorHub*.
+. Enter `node-observability-operator` into the filter box to locate the Node Observability Operator.
+. Click *Install*.
+. On the *Install Operator* page:
+.. *Update channel*: Select *alpha*.
+.. *Installation mode*: Select *A specific namespace on the cluster*.
+.. *Installed Namespace*: Choose *node-observability-operator* from the list.
+.. *Update approval*: Select *Automatic*.
+.. Click *Install*.
+
+== Deploying the Node Observability Operator
+Before you can deploy the Node Observability Operator, you should create a `NodeObservability` custom resource (CR).
```

Comment:
Hi @darshan-nagaraj 
We don't really need `NodeObservability` custom resource in order to deploy the operator. 
The operator will be deployed by the above procedure, and will handle `NodeObservability` CRs once created.
What you can say however, is that `NodeObservability` is needed in order to create a `NodeObservabilityRun`, which is the profiling request.

Reply:
Fixed.